x-pack/plugin/core/src/main/java/org/elasticsearch/license/LicenseService.java

```diff
@@ -72,7 +73,8 @@
      */
     static final TimeValue GRACE_PERIOD_DURATION = days(7);
 
-    public static final long BASIC_SELF_GENERATED_LICENSE_EXPIRATION_MILLIS = Long.MAX_VALUE - days(365).millis();
+    public static final long BASIC_SELF_GENERATED_LICENSE_EXPIRATION_MILLIS =
```

Comment:
I moved this to the client can see the constant. I'd like to keep this constant here for now and move it to `License.java` once `License.java` is moved.